extensions/wikia/ExactTargetUpdates/client/builders/UpdateRequestBuilder.class.php

```diff
@@ -99,4 +118,19 @@ private function prepareUserEditsParams( $edits ) {
 		return $result;
 	}
 
+	private function prepareWikiParams( $wikiData ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "$wikiData" which has the same name as the field declared at line 12. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=php%3AS1117)